larky/src/main/resources/stdlib/parse.star

```diff
@@ -0,0 +1,161 @@
+# This module defines a standard interface to break Uniform Resource Locator (URL)
+# strings up in components (addressing scheme, network location, path etc.)
+
+# Direct copy from: https://github.com/python/cpython/blob/3.9/Lib/urllib/parse.py
```

Comment:
We should move this to `stdlib/urllib` instead of having this in `stdlib`

Reply:
yes, that is my plan, putting it here temporarily for convenience